util/syspolicy/handler_windows.go

```diff
@@ -18,6 +18,36 @@ var (
 
 type windowsHandler struct{}
 
+var stringKeys = []Key{
```

Comment:
No objection, but why this move?

Reply:
When running `staticcheck` from CI, we run with `GOOS=linux` which never sees this file. Since these vars are only used in the windows code, they appeared as unused. Moving them to the windows-specific file was the easiest fix :sweat_smile: 

Unfortunately `staticcheck` cannot account for GOOS-specific code and evaluate if stuff is used under any possible OS.